app/src/androidTestE2eTest/java/io/appium/uiautomator2/unittest/test/internal/Logger.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.appium.uiautomator2.unittest.test.internal;
+
+public class Logger {
```

Comment:
Don't we already have the logger class in UIA2?

Reply:
It's useful to have different tags for server and e2e tests.

Reply:
Then subclass the original class rather than creating a new one

Reply:
I don't like to create dependencies on AUT not for testing purpose, but if you insist - ok.